src/main/java/org/jabref/gui/errorconsole/ErrorConsoleView.java

```diff
@@ -112,6 +114,17 @@ public void updateItem(LogEventViewModel event, boolean empty) {
                     setGraphic(graphic);
                 }
             }
+
+            {
```

Comment:
Why this opening and closing brace below? Should that block go to a separate method?

Reply:
I was defining an instance initializer block for an anonymous inner class that is being created as the cell factory for the ```messagesListView```.
Good point though, that makes it very badly readable, I'll move it to a separate method instead